contract/src/postal-service.contract.js

```diff
@@ -12,16 +12,18 @@ export const meta = harden({
 // compatibility with an earlier contract metadata API
 export const { customTermsShape } = meta;
 
+let issuerNumber = 1;
+const IssuerShape = M.remotable('Issuer');
```

Comment:
import from ertp?

Reply:
Done. I did a deep import because of a comment in the swap contract about minimizing bundle size.